assets/src/scss/styleguide/src/components/_buttons.scss

```diff
@@ -12,23 +12,25 @@
 .btn,
 .wp-block-button a,
 .wp-block-file .wp-block-file__button {
+  --button-- {
+    font-family: $roboto;
+    text-align: center;
+    font-weight: bold;
+    border-radius: 4px;
+    padding: 0 30px;
+    transition-property: color, background-color, border-color;
```

Comment:
These transition related properties are mostly made variable to be able to disable the transition, and not to fine tune it. The transition causes a weird effect for the climate theme, which uses gradient buttons. 
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/3cfd56585552a4e3df8e43ad3e442d2ddc7d5278/assets/src/styles/campaigns/themes/_theme_climate.scss#L22-L25
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/3cfd56585552a4e3df8e43ad3e442d2ddc7d5278/assets/src/styles/campaigns/themes/_theme_climate.scss#L35-L39

Reply:
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/3cfd56585552a4e3df8e43ad3e442d2ddc7d5278/assets/src/styles/campaigns/themes/_theme_climate.scss#L22-L25

Reply:
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/3cfd56585552a4e3df8e43ad3e442d2ddc7d5278/assets/src/styles/campaigns/themes/_theme_climate.scss#L35-L39